### PR TITLE
Docs changes for ebpf ipv6 GA

### PR DESCRIPTION
**Pull request by sridhartigera**
This PR covers the docs changes for eBPF IPV6-GA. 

EPIC - https://tigera.atlassian.net/browse/CORE-9982

Product Version(s):
Calico

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico/operations/ebpf/enabling-ebpf.mdx [hunk 1]
```diff
@@ -51,15 +51,11 @@ eBPF (or "extended Berkeley Packet Filter"), is a technology that allows safe mi
...
-:::note
-This feature is tech preview. Tech preview features may be subject to significant changes before they become GA.
-:::
-
 Limitations:
 
-  - Dual stack is not supported. However, ipv4 traffic is allowed into hosts (not workloads) because many managed Kubernetes environments have ipv4-based control plane.
-  - `doNotTrack` policies (xdp) are not supported
-  - IPIP is not supported ({{prodname}} iptables does not support it either). VXLAN is the recommended overlay for eBPF mode.
+  - IPv6 only hosts are not supported.
```
Comment (ctauchen) [inline calico/operations/ebpf/enabling-ebpf.mdx]: I'm a bit confused about what's supported. Above we say IPv6 is supported, and here we say it isn't. 

Is it better to specify that we support IPv4-only and dual-stack?

File: calico/operations/ebpf/enabling-ebpf.mdx [hunk 2]
```diff
@@ -51,15 +51,11 @@ eBPF (or "extended Berkeley Packet Filter"), is a technology that allows safe mi
...
 - IPv6
 
-:::note
-This feature is tech preview. Tech preview features may be subject to significant changes before they become GA.
-:::
-
 Limitations:
 
-  - Dual stack is not supported. However, ipv4 traffic is allowed into hosts (not workloads) because many managed Kubernetes environments have ipv4-based control plane.
-  - `doNotTrack` policies (xdp) are not supported
```
Comment (ctauchen) [inline calico/operations/ebpf/enabling-ebpf.mdx:61]: Did you intend to remove this? Is this a ticketed item?

File: calico/operations/ebpf/enabling-ebpf.mdx [hunk 3]
```diff
@@ -51,15 +51,11 @@ eBPF (or "extended Berkeley Packet Filter"), is a technology that allows safe mi
...
-  - IPIP is not supported ({{prodname}} iptables does not support it either). VXLAN is the recommended overlay for eBPF mode.
+  - IPv6 only hosts are not supported.
+  - IPIP is not supported ({{prodname}} iptables does not supporte it either). VXLAN is the recommended overlay for eBPF mode.
```
Comment (ctauchen) [inline calico/operations/ebpf/enabling-ebpf.mdx]:
```suggestion
  - IPIP is not supported ({{prodname}} iptables does not support it either). VXLAN is the recommended overlay for eBPF mode.
```

**Review by ctauchen — COMMENTED** [review]
A few comments. Could you make sure to add a link to the ticket for this one? Context is always helpful.

File: calico/operations/ebpf/enabling-ebpf.mdx (hunk 2 above)
Comment (tomastigera) [reply calico/operations/ebpf/enabling-ebpf.mdx:61]: that is correct, we have v6 support for this now

File: calico/operations/ebpf/enabling-ebpf.mdx (hunk 1 above)
Comment (tomastigera) [reply calico/operations/ebpf/enabling-ebpf.mdx]: > Is it better to specify that we support IPv4-only and dual-stack?

that sounds better, I also got confused

File: calico/operations/ebpf/enabling-ebpf.mdx (hunk 2 above)
Comment (ctauchen) [reply calico/operations/ebpf/enabling-ebpf.mdx:61]: Is line 61 related to IPv6?

Comment (tomastigera) [reply calico/operations/ebpf/enabling-ebpf.mdx:61]: This whole block is ipv6, the feature preview note above made it a bit more difficult to see that. I noticed that. But that is all going away.

File: calico/operations/ebpf/enabling-ebpf.mdx (hunk 1 above)
Comment (tomastigera) [reply calico/operations/ebpf/enabling-ebpf.mdx]: we are working on this limitation to go away

Comment (ctauchen) [reply calico/operations/ebpf/enabling-ebpf.mdx]: Ok, let's change as follows, starting with line 52:

```
- IPv4-only and dual-stack hosts

Limitations:

- IPv6-only hosts are not supported.

File: calico/operations/ebpf/install.mdx [hunk 4]
```diff
@@ -51,15 +51,10 @@ and in particular, pushing the networking capabilities of the latest Linux kerne
...
-:::
-
 Limitations:
-
-  - Dual stack is not supported. However, ipv4 traffic is allowed into hosts (not workloads) because many managed Kubernetes environments have ipv4-based control plane.
-  - `doNotTrack` policies (xdp) are not supported
-  - IPIP is not supported ({{prodname}} iptables does not support it either). VXLAN is the recommended overlay for eBPF mode.
+  
+  - IPv6 only hosts are not supported.
+  - IPIP is not supported ({{prodname}} iptables does not supporte it either). VXLAN is the recommended overlay for eBPF mode.
```
Comment (ctauchen) [inline calico/operations/ebpf/install.mdx]:
```suggestion
  - IPIP is not supported ({{prodname}} iptables does not support it either). VXLAN is the recommended overlay for eBPF mode.
```

File: calico/operations/ebpf/install.mdx [hunk 5]
```diff
@@ -51,15 +51,10 @@ and in particular, pushing the networking capabilities of the latest Linux kerne
...
-This feature is tech preview. Tech preview features may be subject to significant changes before they become GA.
-:::
-
 Limitations:
-
-  - Dual stack is not supported. However, ipv4 traffic is allowed into hosts (not workloads) because many managed Kubernetes environments have ipv4-based control plane.
-  - `doNotTrack` policies (xdp) are not supported
-  - IPIP is not supported ({{prodname}} iptables does not support it either). VXLAN is the recommended overlay for eBPF mode.
+  
+  - IPv6 only hosts are not supported.
```
Comment (ctauchen) [inline calico/operations/ebpf/install.mdx]: See suggested text above.

**Review by ctauchen — COMMENTED** [review]
A few small things and we're done.

**Comment by ctauchen** [comment]
LGTM, thanks @sridhartigera

**Pull request merged**